Makefile

```
@@ -9,19 +9,25 @@ OS=$(shell uname)
 
 ifeq ($(OS), Darwin)
     ARCH=$(shell uname -m)
-    LLVM_CONFIG=llvm-config
 
     # LLVM Version Setting  
     LLVM_VERSION_PATTERN="^11\."
     LLVM_VERSION="11"
     ifeq ($(ARCH), arm64)
-	LLVM_VERSION="13"
+        LLVM_VERSION="13"
         LLVM_VERSION_PATTERN="^13"
```

Can we add the fix for the pattern too? 
```suggestion
        LLVM_VERSION_PATTERN="^13\."
```

---

Sure thing. Just committed the suggestion (and made a small aesthetic change, swapping two of the prior lines to match the later lines).